config/ember-try.js

```diff
@@ -39,6 +39,20 @@ module.exports = {
           'ember': 'canary'
         }
       }
+    },
+    {
+      name: 'ember-source',
+      allowedToFail: true,
+      bower: {
+        dependencies: {
+          'ember': null
+        }
+      },
+      npm: {
+        devDependencies: {
+          'ember-source': '^2.11.0-alpha.1'
```

Comment:
@rwjblue I might have been a little quick on the merge. Does this need to also be in the `package.json` too?